tools/syz-declextract/syz-declextract.cpp

```diff
@@ -54,7 +63,7 @@ class EnumMatcher : public MatchFinder::MatchCallback {
     const auto value = int(*enumValue->getAPValueResult().getInt().getRawData());
     const auto &path = std::filesystem::relative(
         Result.SourceManager->getFilename(enumValue->getEnumConstantDecl()->getSourceRange().getBegin()).data());
-    EnumDetails.push_back({std::move(name), value, std::move(path)});
+    EnumDetails[value] = {std::move(name), std::move(path)};
```

Comment:
It is the actual enum value. It was in case there were any unnamed attributes,I now handle this in the netlink function itself to keep the matcher generic.